apps/services/auth/public-api/README.md

```diff
@@ -0,0 +1,7 @@
+<!-- gitbook-navigation: "Public API" -->
```

Comment:
@johannaagma is also adding readme to the auth public api on how to start the db for running locally in #8354. So we just need to be aware of that and merge the changes.